ops/cli.py

```diff
@@ -275,6 +295,32 @@ def print_deployment_success(config: WflInstanceConfig) -> None:
         run_liquibase_migration,
         stop_cloud_sql_proxy,
         print_deployment_success
+    ],
+    "ops-deploy": [
```

Comment:
How about changing this to something like "deploy-tag" or "deploy-from-tag"? It's not immediately clear what "ops-deploy" means